<launch>

	<!-- display the rois GroundPlane 
  <node pkg="opt_utils" type="roi_viewer" name="PeopleDetectionViewer" output="screen">
    <param name="mode" type="string" value="roi_display"/> 
    <param name="label" type="int" value="1" />
    <param name="show_confidence" type="bool" value="true" />
    <remap from ="input_image" to="camera/rgb/image_raw"/>
	<remap from ="input_detections" to="ground_based_people_detector/detections"/>  
  </node>
	-->

	<!-- display the rois HaarDispAda -->
   <node pkg="opt_utils" type="roi_viewer" name="HaarDispAdaViewer" >
     <param name="mode" type="string" value="roi_display"/> 
     <param name="label" type="int" value="1" />
     <param name="show_confidence" type="bool" value="true" />
	 <remap from ="input_image" to="HaarDispAdaColorImage"/>
	 <remap from ="input_detections" to="detector/detections"/>  
   </node>

	<!-- display the roi target-->
   <node pkg="opt_utils" type="target_viewer" name="TargetViewer" >
     <param name="mode" type="string" value="roi_display"/> 
	 <remap from ="input_image" to="HaarDispAdaColorImage"/>
	 <remap from ="input_track" to="follower/tracker/target"/>  
   </node>


</launch>
